src/coreclr/vm/eeconfig.cpp

```diff
@@ -943,6 +942,52 @@ TypeNamesList::TypeNamesList()
     LIMITED_METHOD_CONTRACT;
 }
 
+namespace
+{
+    //
+    // Slight modification of Rob Pike's minimal regex from The Practice of Programming.
+    // See https://www.cs.princeton.edu/~bwk/tpop.webpage/code.html - 'Grep program from Chapter 9'
```

Comment:
This should get mention in https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT. Something like:
```
License notice for code from The Practice of Programming
-------------------------------

Copyright (C) 1999 Lucent Technologies

Excerpted from 'The Practice of Programming
by Brian W. Kernighan and Rob Pike

You may use this code for any purpose, as long as you leave the copyright notice and book citation attached.
```